attributes/default.rb

```diff
@@ -129,6 +128,11 @@
 
 default['kube-hops']['serving_node_labels']                = ""
 default['kube-hops']['serving_node_tolerations']           = ""
+default['kube-hops']['serving_max_memory_allocation']      = "-1"   # no upper limit
```

Comment:
Does this mean a serving can allocate an unlimited number of MBs? Would it not be better to have an upper limit by default? For example the slider in the UI to configure the memory needs to know the maximum right?

Reply:
Same applies to cores.

Reply:
A value of -1 means that the user could set any resource limit for a deployment. However, we also have default values for request limits so it won't allocated unlimited resources by default.